lib/credit_card_sanitizer.rb

```diff
@@ -59,7 +60,7 @@ def initialize(options = {})
   # Returns a String of the redacted text if a credit card number was detected.
   # Returns nil if no credit card numbers were detected.
   def sanitize!(text)
-    to_utf8!(text)
+    text.scrub!('�')
```

Comment:
They are not the same. This gem is using the replacement char (unicode 0xFFFD), while scrub_rb was using `?` (unicode 0x3F)